translation.json

```diff
@@ -58,6 +58,9 @@
         "defaultOptions": {
           "option1": "Option 1",
           "option2": "Option 2"
+        },
+        "isSelectAll": {
```

Comment:
This is incorrect, see https://github.com/Fliplet/fliplet-widget-form-builder/pull/451/files#r923398490